src/lib/fetchTickerData.ts

```diff
@@ -56,18 +56,15 @@ export const addTickerDataToSettings = (tickerSettings: TickerSettings): Promise
         tickerData: tickerData,
     }));
 
-export const addTickerDataToVariant = (variant: Variant): Promise<Variant> => {
+export const getTickerSettings = (variant: Variant): Promise<TickerSettings | undefined> => {
```

Comment:
Refactored to match return the `tickerSettings` as apposed to updating the variant to be more uniform with how we handle reminders